src/components/PopoverListItem/PopoverListItem.module.css

```diff
@@ -0,0 +1,32 @@
+/*------------------------------------*\
+    # POPOVER LIST ITEM
+\*------------------------------------*/
+
+/**
+ * PopoverListItem
+ */
+.popover-list-item {
+  display: flex;
+  font-size: var(--eds-font-size-14);
+  padding: var(--eds-size-1) var(--eds-size-2);
+  background-color: var(--eds-theme-color-form-background);
+  cursor: pointer;
+}
+
+.popover-list-item--active {
+  background-color: var(--eds-theme-color-background-neutral-default-hover);
+}
+
+.popover-list-item--disabled {
+  color: var(--eds-theme-color-text-disabled);
+  pointer-events: none;
```

Comment:
Is this necessary? Looks like you're not passing in any interactive props into the item if it's disabled, so what pointer events are you removing? `cursor: not-allowed;` might be good though; that's what we use on disabled buttons.

Reply:
good catch. an earlier version of this included an anchor which did have behaviors, and this prevented bubbling up to that to trigger actions.

The cursor is undefined from design side, but I can add that for consistency, and remove this `pointer-events` property.